pinc/project_quick_check.inc

```diff
@@ -41,19 +42,18 @@ function _test_project_for_large_page_images($projectid)
         $details .= "<th>" . _("Size") . "</th>";
         $details .= "</tr>";
 
-        $page_image_size_limit = 100; // kb
-
         $num_large_images = 0;
         while(list($image) = mysqli_fetch_row($result_res)) {
             // the the file's info
             $file_info = get_file_info_object($image,"$projects_dir/$projectid","$projects_url/$projectid");
 
-            // check to see if the file is > $page_image_size_limit kb
-            if($file_info->size > ($page_image_size_limit * 1024))
+            // check to see if the file is > size limit
+            $error = get_image_size_error($file_info->size);
+            if(isset($error))
```

Comment:
You want `if($error)` here. If `$error` is NULL, this will still be true. PHP is weird.

Reply:
And I am wrong!
> isset — Determine if a variable is declared and is different than NULL

From https://www.php.net/manual/en/function.isset.php

You can leave this as-is :)

Reply:
Yeah I didn't want to rely on a falsy check, so I looked up isset seems to be the best way to check for not null to me.